prog/main.go

```diff
@@ -274,6 +275,7 @@ func setupFlags(flags *flags) {
 	flag.Bool("app-only", false, "Only run the app.")
 
 	// Probe flags
+	flag.BoolVar(&flags.probe.printOnStdout, "probe.print-on-stdout", false, "Print reports on stdout instead of sending to app, for debugging")
```

Comment:
`probe.publish.stdout`? (we already have a `probe.publish.interval`).

Reply:
also, perhaps the flags processing code should warn/fail when this is set and the computed `targets` is non-empty.